system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -207,6 +207,24 @@
         display: none;
       }
     }
+
+    &.dragged {
+      .tile {
+        background: $grey-20;
+        box-shadow: none;
+
+        &::before,
+        .screenshot,
+        .rich-icon,
+        .top-site-icon {
+          display: none;
```

Comment:
Not sure if it would be better to have the component just render fewer items, e.g., just a `.tile` with no contents? Seems like this approach could easily result in structure render changes to accidentally appear when dragged. Or would a `*` descendant match be too much? Hmm...